pkg/controller/postgres/postgres_controller.go

```diff
@@ -176,6 +177,14 @@ func (r *ReconcilePostgres) Reconcile(request reconcile.Request) (reconcile.Resu
 			return reconcile.Result{Requeue: true, RequeueAfter: p.GetReconcileTime(instance)}, nil
 		}
 
+		if annotations.Has(instance, annotations.Created, annotations.Created) {
+			r.logger.Info("'created' annotation found, skipping postgres reconcile")
```

Comment:
I lean more towards as much logging as possible. Looking at the logging that has been done in [cluster service](https://github.com/integr8ly/cluster-service/pull/3/files#diff-5d6e6baf22f72630659992325db5e91dR44) I think we should adopt this approach, and look to refactoring the logging. 

So in short for now, I would leave in as much logging as possible, allowing us to come back to refactor. Just my two cents

Reply:
> Logs consume pod memory

@grdryn I'm interested in the rationale behind this comment.
I'm not holding up the PR or suggesting any changes, but I would like to know if there's more to this than my simplistic understanding.
I would think writing logs takes up some nominal amount of cpu & memory while writing, with most resource usage being filesystem for writing them somewhere. If a container is spewing out logs continuously at a rapid pace, I would understand an increase in memory usage, but nothing in the realm of anything significant so as to get the pod killed. If anything, cpu & storage would be the things that ramp up in usage.

Am I missing something?

Reply:
@david-martin I could be wrong about it, but we hit memory limits after a couple of days in the UPS operator because of the spammy logs (caused by an amq-online issue). I guess it could be due to something else, but our determination at the time was that it was holding them in memory and that's what was using all of the memory. It wasn't a big issue really...it just meant that the pod was recycled more frequently than we were expecting.

@ciaranRoche @david-martin the real reason I bring it up here is that this is the "normal" case, where the initial creation is a "special" case. Even for the happy path, these CRO controllers requeue a new reconcile request every <duration> (`p.GetReconcileTime(instance)`). That means that there will be a bunch of these "nothing to see here" messages (at least one per CR per <duration>). That's probably fine, and I'm probably just rambling. :slightly_smiling_face: